### DISCUSSION TITLE
How to wake screen back up

### DISCUSSION THREAD

**Discussion by jrose78** [KlipperScreen/KlipperScreen#700] [Q&A]
I have a pi 7" touchscreen and everything works great except when the screen goes to sleep I can't wake it back up. I figured I was missing something and did a couple searches on how to wake the screen back up, but I didn't find an answer so I figured I would post here. I have a pi 4 and the 7" touchscreen. I have of course tried touching the screen until it started to feel uncomfortable.  I hit the power button and turned it off and back on but it say no signal.  Any ideas?

Thank you!

Edit: Just figured out that if I power off the monitor and then power it back on when it says no input and I tap the screen it comes back to life.  So when the screen is off it does not register touches.

**Comment by blonestar** [comment]
Having the same issue with BigTreeTech PI TFT50 + PI4 (2GB), unable to wake the screen up.
Sometimes I succeed turning it on by restarting KlipperScreen service only from the Mainsail, but sometimes that does not work.
I'll try to roll the wheel when it happens next time like @jrose78 has mention.
Also this issue start happening not so long time ago, a month or less.

**Comment by blonestar** [comment]
I can confirm issue is still there and I'll create an issue.
I succeed to wake up the screen by restarting KlipperScreen service from the Mainsail but that's not always possible.
PI4 & Display are connected via DSI.

**Comment by alfrix** [comment]
https://github.com/KlipperScreen/KlipperScreen/commit/c09e552e3ac348643e0abaedfbf2ea0bf471f952
